### PR TITLE
Fix delete

### PR DESCRIPTION
**Pull request by armannaj**

### CODE REVIEW

File: PurpleExplorer/Helpers/ServiceBusHelper.cs [hunk 1]
```diff
@@ -99,35 +101,31 @@ public async Task SendTopicMessage(string connectionString, string topicPath, st
             await topicClient.CloseAsync();
         }
 
-        public void DeleteMessage(string connectionString, string topicPath, string subscriptionPath,
+        public async void DeleteMessage(string connectionString, string topicPath, string subscriptionPath,
```
Comment (yjlim214) [inline PurpleExplorer/Helpers/ServiceBusHelper.cs:104]: Should we return `Task`?
```suggestion
        public async Task DeleteMessage(string connectionString, string topicPath, string subscriptionPath,
```

File: PurpleExplorer/ViewModels/MainWindowViewModel.cs [hunk 2]
```diff
@@ -180,19 +183,21 @@ public async void DeleteMessage()
...
+            // Because buttonResult can be None or No
+            if (buttonResult != ButtonResult.Yes)
             {
                 CurrentMessage = null;
                 return;
             }
 
+            Log($"Deleting message {_currentMessage.MessageId}...");
             var connectionString = CurrentTopic.ServiceBus.ConnectionString;
             _serviceBusHelper.DeleteMessage(connectionString, _currentTopic.Name, _currentSubscription.Name,
```
Comment (yjlim214) [inline PurpleExplorer/ViewModels/MainWindowViewModel.cs:197]: Should we `await`?
```suggestion
            await _serviceBusHelper.DeleteMessage(connectionString, _currentTopic.Name, _currentSubscription.Name,
```

File: PurpleExplorer/ViewModels/MainWindowViewModel.cs [hunk 3]
```diff
@@ -180,19 +183,21 @@ public async void DeleteMessage()
...
                 _currentMessage, _currentMessage.IsDlq);
             CurrentMessage = null;
-            Log("Message deleted.");
+            Log("Message deleted (might take some seconds to reflect)");
```
Comment (yjlim214) [inline PurpleExplorer/ViewModels/MainWindowViewModel.cs:200]: with the `await` changes, the `Message deleted` log will only appear when it has been deleted. So maybe we could add `(might take some seconds to reflect)` to the `Deleting message` log?

Comment (armannaj) [reply PurpleExplorer/ViewModels/MainWindowViewModel.cs]: good idea

File: PurpleExplorer/ViewModels/MainWindowViewModel.cs (hunk 2 above)
Comment (armannaj) [reply PurpleExplorer/ViewModels/MainWindowViewModel.cs]: yes, we should. I changed it.

File: PurpleExplorer/Helpers/ServiceBusHelper.cs (hunk 1 above)
Comment (armannaj) [reply PurpleExplorer/Helpers/ServiceBusHelper.cs]: Done. thanks.

**Review by yjlim214 — APPROVED** [review]
Looks good!

**Pull request merged**